pandas/tests/test_frame.py

```diff
@@ -5545,6 +5545,8 @@ def test_itertuples(self):
         dfaa = df[['a', 'a']]
         self.assertEqual(list(dfaa.itertuples()), [(0, 1, 1), (1, 2, 2), (2, 3, 3)])
 
+        self.assertEqual(type(next(df.itertuples(name=None))), tuple)
```

Comment:
- If you want to test if an instance is a namedtuple use `is_named_tuple` 
- If you want to test if an instance is a specific namedtuple, I would check the `repr` matches the `repr` you expect. Also OK if you additionally check the instances' equality / if the expected instance is a namedtuple.